e2e/cypress/support/api_commands.js

```diff
@@ -979,6 +979,24 @@ Cypress.Commands.add('uninstallPluginById', (pluginId) => {
     });
 });
 
+/**
+ * Get all user`s plugins.
+ *
+ */
+Cypress.Commands.add('getAllPlugins', () => {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/plugins',
+        method: 'GET',
+        failOnStatusCode: false,
+    }).then((response) => {
+        if (response.status !== 200 && response.status !== 404) {
+            expect(response.status).to.equal(200);
+        }
```

Comment:
I'm confused with this conditional block.  Seems like the inside `expect` will not be hit.  Can the conditional be removed and just retain the `expect(response.status).to.equal(200);` since we are expecting OK status in getting all plugins?  Otherwise, fail the test and don't continue.